fix: Update declare_a_smart_contract.adoc (#1077)
starknet-io/starknet-docs: components/Starknet/modules/quick_start/pages/declare_a_smart_contract.adoc

[TIP]
====

> If you require an example smart contract for testing, you can use this link:https://github.com/starknet-edu/starknetbook/blob/main/examples/vote-contracts/src/lib.cairo[example from the Starknet Book].

Reviewer: ```suggestion
If you require a smart contract for testing, you can use this sample contract, link:https://github.com/starknet-edu/starknetbook/blob/main/examples/vote-contracts/src/lib.cairo[`lib.cairo`], from the Starknet Book].
```

> If you require a smart contract for testing, you can use this sample contract, link:https://github.com/starknet-edu/starknetbook/blob/main/examples/vote-contracts/src/lib.cairo[`lib.cairo`], from the Starknet Book].

Reviewer: ```suggestion
If you require a smart contract for testing, you can use this sample contract, link:https://github.com/starknet-edu/starknetbook/blob/main/examples/vote-contracts/src/lib.cairo[`lib.cairo`], from the Starknet Book.
```

> If you require an example smart contract to test with, you can use a pre prepared example from the Starknet Book link:https://github.com/starknet-edu/starknetbook/blob/main/examples/vote-contracts/src/lib.cairo[here].

Reviewer: ```suggestion
If you require an example smart contract for testing, you can use this link:https://github.com/starknet-edu/starknetbook/blob/main/examples/vote-contracts/src/lib.cairo[example from the Starknet Book].
```
